.env.secrets.template

```diff
@@ -0,0 +1,6 @@
+NACHET_BLOB_ACCOUNT=""
+NACHET_BLOB_KEY=""
+NACHET_BLOB_URL=""
+NACHET_DB_URL=""
```

Comment:
Do you think it would be possible to store both the username and password instead of the full URL for the `.env.secrets.template` file? Full url's are quite hard to manage with the git-secrets tool